crunchydb/high-availablility/PostgresCluster.yaml

```diff
@@ -5,13 +5,39 @@ metadata:
 spec:
   monitoring:
     pgmonitor:
+      # this stuff is for the "exporter" container in the "hippo-ha-pgha1" set of pods
       exporter:
         image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.0.4-0
+        resources:
+          requests:
+            cpu: 50m
+            memory: 64Mi
+          limits:
+            cpu: 100m
+            memory: 256Mi
   image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.5-0
   postgresVersion: 13
   instances:
     - name: pgha1
       replicas: 3
+      # these resources are for the "database" container in the "hippo-ha-pgha1" set of pods
+      resources:
+        requests:
+          cpu: 50m
+          memory: 64Mi
+        limits:
+          cpu: 250m
+          memory: 512Mi
+      sidecars:
+        # this stuff is for the "replication-cert-copy" container in the "hippo-ha-pgha1" set of pods
+        replicaCertCopy:
+          resources:
+            requests:
+              cpu: 50m
+              memory: 32Mi
+            limits:
+              cpu: 100m
+              memory: 64Mi
       dataVolumeClaimSpec:
         accessModes:
         - "ReadWriteOnce"
```

Comment:
The storage request below should be lowered to fit in the extra-small namespaces